pandas/tests/io/parser/test_encoding.py

```diff
@@ -302,13 +302,17 @@ def test_readcsv_memmap_utf8(all_parsers):
     tm.assert_frame_equal(df, dfr)
 
 
-def test_not_readable(all_parsers):
+@pytest.mark.parametrize("mode", ["w+b", "w+t"])
+def test_not_readable(all_parsers, mode):
     # GH43439
     parser = all_parsers
-    if parser.engine in ("python", "pyarrow"):
-        pytest.skip("SpooledTemporaryFile does only work with the c-engine")
-    with tempfile.SpooledTemporaryFile() as handle:
-        handle.write(b"abcd")
+    if parser.engine == "pyarrow":
+        pytest.skip("SpooledTemporaryFile does only work with the pyarrow-engine")
```

Comment:
can you make this an xfail?  ive been finding many of our skips have been silently fixed